src/utils.js

```diff
@@ -18,6 +18,22 @@ genotet.utils.PIXEL_TOLERANCE = .1;
 /** @const {number} */
 genotet.utils.RANGE_TOLERANCE = .001;
 
+/** @const {RegExp} */
+genotet.utils.VALIDATE_EMAIL_REGEX =
+  /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+
+/** @const {RegExp} */
+genotet.utils.VALIDATE_USERNAME_REGEX = /^\w+$/;
+
+/** @const {RegExp} */
+genotet.utils.VALIDATE_PASSWORD_REGEX = /^\w+$/;
```

Comment:
validate is a verb. You can just say PASSWORD_REGEX or VALID_PASSWORD_REGEX (more verbose).